src/hooks/useViewState.ts

```diff
@@ -137,15 +137,17 @@ export function useViewState({
            * very difficult to interact with. We create an element with maximum focus priority and
            * focus it so that when we delete it the user will be at the start of the focus order
            * just like if they had freshly loaded the page. */
-          const shouldFocus = autofocus !== 'off';
-          if (savedActiveElement !== null && shouldFocus) {
-            savedActiveElement.focus();
-          } else {
-            const tempInteractiveEl = document.createElement('span');
-            tempInteractiveEl.setAttribute('tabindex', '1');
-            document.body.prepend(tempInteractiveEl);
-            tempInteractiveEl.focus();
-            tempInteractiveEl.remove();
+          const shouldAutoFocus = autofocus !== 'off';
+          if (shouldAutoFocus) {
+            if (savedActiveElement !== null) {
+              savedActiveElement.focus();
+            } else {
+              const tempInteractiveEl = document.createElement('span');
+              tempInteractiveEl.setAttribute('tabindex', '1');
```

Comment:
no need for `setAttribute` afaik

```suggestion
              tempInteractiveEl.tabIndex = 1;
```